### PR TITLE
README: Add info about DaytL

### PR DESCRIPTION
**Pull request by zillar**
README: Add info about DaytL

### CODE REVIEW

File: Members/DaytL.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## About
+Hello, I am Rishabh Raghuvanshi, currently pursuing B.Tech in Information Technology.
+
+## Place
+Indore, M.P
+
+## Programming Languages: 
+ -C++
```
Comment (CapTen101) [inline Members/DaytL.md]: @DaytL  the syntax for bullet points should be:
`- C++`
`- Java`
`- Python`

**Review by CapTen101 — CHANGES_REQUESTED** [review]

File: Members/DaytL.md (hunk 1 above)
Comment (DaytL) [reply Members/DaytL.md]: I have updated the Syntax.

Comment (zillar) [reply Members/DaytL.md]: Thanks for feedback. I have resolved the issue.

Comment (CapTen101) [reply Members/DaytL.md]: great

**Comment by CapTen101** [comment]
Also can you squash your commits into one? That way our commit history stays plain and simple. Read this [guide](https://www.internalpointers.com/post/squash-commits-into-one-git) for that. @DaytL

**Comment by zillar** [comment]
> 
> 
> Also can you squash your commits into one? That way our commit history stays plain and simple. Read this [guide](https://www.internalpointers.com/post/squash-commits-into-one-git) for that. @DaytL

That's an cool feature. I will keep that in mind. Thx for sharing.
Hey, but the problem is the second commit I did was from browser(Github) and hence its not in my git log on PC so I couldn't squash it there. Is there any other method left other than making a new pull request?

**Comment by CapTen101** [comment]
@DaytL See, It doesn't matter from where you committed your work. If you have committed from github online, then `pull` those changes from your forked repository onto your local repository. If you've committed from your local machine then you anyways have to `push` your changes to your forked repository.

Hence in your case, just `pull` those changes you did from your github onto our local repository by:
`git pull origin <branch-name>`. Here you've named your branch by your username `DaytL`, next time try to give a meaning full name related to the work you intend to do on that branch. It's easier that way to assess the pull request and is a good practice :slightly_smiling_face:

**Review by CapTen101 — COMMENTED** [review]
@DaytL See, It doesn't matter from where you committed your work. If you have committed from github online, then `pull` those changes from your forked repository onto your local repository. If you've committed from your local machine then you anyways have to `push` your changes to your forked repository.

Hence in your case, just `pull` those changes you did from your github onto your local repository by:
`git pull origin <branch-name>`. Here you've named your branch by your username `DaytL`, next time try to give a meaning full name related to the work you intend to do on that branch. It's easier that way to assess the pull request and is a good practice :slightly_smiling_face:

**Comment by zillar** [comment]
@CapTen101 After pulling the changes I was able to see both the commits. Now I have to squash both the commits, so I used command `git rebase --interactive HEAD~[2]` according to link of guide you provided, but its giving an error `fatal: invalid upstream 'HEAD~[2]'`, After googling it I still don't understand what's wrong. So instead I ended up using `git rebase --interactive HEAD~2` and have successfully squashed recent 2 commits. And about the name thing I do think its irrelevant but it was instructed(on README.md  The-Codesis/codesis-yellow-pages/).  
I appreciate you really putting so much effort. Thanks

**Comment by zillar** [comment]
@CapTen101  Do I need to create another pull request because on this page pull showing **DaytL wants to merge 2 commits into `The-codesis:master` from `DaytL:DaytL`**

**Comment by CapTen101** [comment]
> @CapTen101 Do I need to create another pull request because on this page pull showing **DaytL wants to merge 2 commits into `The-codesis:master` from `DaytL:DaytL`**

@DaytL The PR is still showing 2 commits maybe because you haven't pushed them yet? `git push` only will reflect them here.
Just try `git push origin DaytL`

**Comment by zillar** [comment]
@CapTen101 I have checked `git log` and its showing only one commit and on my GitHub repo DaytL/codesis-yellow-pages its also showing only one commit ahead. And I have also tried `git push origin DaytL`. I think I have to make an another pull request. What do you think?

**Comment by CapTen101** [comment]
@DaytL Have you checked your DaytL branch on github? Maybe you might have checked your master branch since it's the default

**Comment by zillar** [comment]
@CapTen101 Damn, you are right!

**Comment by CapTen101** [comment]
It often happens when we are just beginning 😆

**Comment by zillar** [comment]
@CapTen101 BTW I forced push the changes `git push -f master:DaytL` because DaytL branch was ahead of master branch hence `git push master:DaytL` wasn't working. Is there is any other way to do it? Because I just overwritten a branch which is not a problem now but might be problematic in future somewhere.

**Comment by CapTen101** [comment]
@DaytL  Sadly this is how things work when you've rebased your work (there are many `rebase` implementation, one is squashing commits --> our case). Force pushing is the only option and yes it's sometimes risky to `rebase` your work and hence should always be done with caution. I remember I messed up a particular PR back when I was just getting started in contributing through github.

**Review by CapTen101 — APPROVED** [review]
Changes nicely done :+1:

**Pull request merged**